### ISSUE TITLE
Can't Save Decklists

### ISSUE THREAD

**Issue by kyloten** [fafranco82/swdestinydb#237]
I just spent an hour perfecting a decklist and I went to save it, but when I hit save it just went to a blank white screen. When I reloaded, the whole decklist was gone. This has happened several times. I can't build decks anymore. Is there anything I can do about this?

**Comment by fafranco82** [comment]
It's a little bit strange... so, you can't edit o build new decks now?  What is your user account?

**Comment by kyloten** [comment]
That's right. I can edit and I can build new ones, but as soon as I hit save, they're gone. When I visit my decks page, it'll show just "hero deck" as the title and show the "not enough cards" alert. So it knows I started a deck, it just doesn't save any of my progress. This has only been happening for a little while, maybe the past week or two.
My account name is kyloten.

**Comment by kyloten** [comment]
Actually, I just made a deck and it worked fine. So maybe it's solved.

**Comment by kyloten** [comment]
So I think I've figured it out: the decks that aren't working are the ones that have forbidden cards in them. When I use the Finn card that permits red villain weapons and vehicles in my deck, the system doesn't know that so it says those cards are forbidden. Whenever I try to save a deck with forbidden cards, it doesn't work.

**Comment by kdmara** [comment]
Is there any solution to this? 
I had the same problem a couple days ago and decided to try again later. Tried again today, same problem. So I came here and found this.

Same problem trying to save a Finn deck with Red Villain weapons in it.

**Comment by kdmara** [comment]
> @kdmara are you still having this problem? I am. There are certain decks I can save and certain decks I can't.

Yup. Just tried again. I can edit and save my other decklists, but not the one with Finn and Red Villain Weapons. I get an http error 5000 I think
